VOID/dockers/success.py

```diff
@@ -40,6 +44,61 @@ def dock(self, attempts):
 
             if self.fitness(cpx) >= 0:
                 print(f"{trial + 1} attempts to success")
+                cpx = self.rescale(cpx)
                 return [cpx]
 
         return []
+
+    def rescale(self, complex):
+        """Rescale the complex to the 0-1 range so results can be visualized in direct and xyz format.
+
+        Args:
+            complex (Complex): The host-guest complex object containing the host and guest molecules.
+
+        Returns:
+            Complex: The rescaled host-guest complex object.
+        """
```

Comment:
Just to make sure that I'm understanding correctly: this function is translating Cartesian coordinates to fractional coordinates using Pymatgen's `Structure` class?

Reply:
I knew this might bring some trouble. The reason for the use of this function is that the software works as a periodic crystal, but for every iteration the coordinates of the guest are displaced over the previous guest coordinates, so it was generating a correct pose but with an unvisualizable format (see example below). Though for running later calculations this shouldn't matter, I introduced this function that rescales the output once the optimal pose is achieved and allows visualization inside and outside htvs by different softwares.

Output without reescaling function:
Host coords:       1  1  1 
                            2  2  2
                            3   3   3
Guest coords:  235 432 274
                         123  456 789

Reply:
Yes, I think I've seen issues like what you describe. When you pull up the structure in VESTA or ASE (for example), the molecules are far outside the unit cell that is being shown. And this function will translate those positions to the fractional coordinates, right?

Reply:
Yes, it is only translating the molecule from where it ends up outside the unit cell range to where it is placed in the unit cell range. Not changing the result of the docked position achieved at all and allowing visualization.